tools/ci/testrun/script/test_open_posix/test_openposix_.py

```diff
@@ -6984,7 +6984,7 @@ def test_ltp_interfaces_pthread_rwlock_timedrdlock_5_1(p):
 def test_ltp_interfaces_sem_init_6_1(p):
     ret = p.sendCommand(
         "ltp_interfaces_sem_init_6_1",
-        ["PASSED", "passed", "Passed", "PASS"],
+        ["PASSED", "passed", "Passed", "PASS", "skipped"],
```

Comment:
why skip

Reply:
if SEM_VALUE_MAX >= INT_MAX, this case will skipped.
int main(void)
{
	sem_t mysemp;
	int counter = SEM_VALUE_MAX;

	if (SEM_VALUE_MAX >= INT_MAX) {
		puts("Test skipped");
		return PTS_PASS;
	}

	++counter;
	sem_init(&mysemp, 0, counter);

	if (errno == EINVAL) {
		puts("TEST PASSED");
		return PTS_PASS;
	} else {
		puts("TEST FAILED");
		return PTS_FAIL;
	}
}